ui/push-health/helpers.js

```diff
@@ -6,3 +6,15 @@ export const resultColorMap = {
   done: 'info',
   'in progress': 'secondary',
 };
+
+export const filterTests = (tests, filterStr) => {
+  const filters = filterStr.split(' ').map(filter => new RegExp(filter));
+
+  return tests.reduce(
+    (acc, test) =>
+      filters.every(filter => filter.test(test.testName))
+        ? [...acc, test]
+        : acc,
+    [],
+  );
+};
```

Comment:
Instead of that `reduce`, try this:

return tests.filter(test => filters.every(f => f.test(test.testName)));

Reply:
So true!  Thanks for the tip.  Fixed...  :)